core/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreTests.java

```diff
@@ -90,6 +90,7 @@ public void basicWorkFlowTest() throws Exception {
                 .setType("fs").setSettings(Settings.settingsBuilder()
                         .put("location", randomRepoPath())
                         .put("compress", randomBoolean())
+                        .put("compress", false)
```

Comment:
This doesn't make sense with the above setting? Randomize and then hardcode false?